manticore/platforms/evm.py

```diff
@@ -1832,6 +1841,10 @@ def GASLIMIT(self):
         """Get the block's gas limit"""
         return self.world.block_gaslimit()
 
+    def CHAINID(self):
+        """Get current chainid."""
+        return 0
```

Comment:
Should we return `1` by default? It's the value of mainet: https://chainid.network/

Reply:
fixed